cmd/api/main.go

```diff
@@ -2,81 +2,100 @@ package main
 
 import (
 	"context"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+
 	"github.com/joho/godotenv"
+	"github.com/kardiachain/kardia-explorer-backend/kardia"
+	"github.com/kardiachain/kardia-explorer-backend/server/api"
 	"go.uber.org/zap"
 
-	"github.com/kardiachain/kardia-explorer-backend/api"
+	kClient "github.com/kardiachain/go-kaiclient/kardia"
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>could not import github.com/kardiachain/go-kaiclient/kardia (-: could not load export data: no export data for "github.com/kardiachain/go-kaiclient/kardia") (typecheck)